presto-base-arrow-flight/src/test/java/com/facebook/plugin/arrow/TestArrowFlightQueriesWithDictionaryVector.java

```diff
@@ -84,7 +84,7 @@ public void testDictionaryBlock()
         // Validate the row count first
         assertEquals(actual.getRowCount(), 3);
         // Now, validate each row
-        MaterializedResult expectedRow = resultBuilder(getSession(), createVarcharType(10), BigintType.BIGINT)
+        MaterializedResult expectedRow = resultBuilder(getSession(), VarcharType.VARCHAR, BigintType.BIGINT)
```

Comment:
why would `createVarcharType(10)` not work?

Reply:
The Field which we created doesn't have precision , the default data from tpch data had varchar(10) but now we have returned only varchar

Reply:
can you import static `VarcharType.VARCHAR`